src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public Ball getRandomBall() {
+        Random randomBall = new Random();
+        ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
The same comment applies to the `ColorSupplier` instance. It would be better to have one instance of `ColorSupplier` as a field in the `Lottery` class and use it in this method.